lms/envs/common.py

```diff
@@ -349,11 +349,8 @@
     # Show Language selector.
     'SHOW_LANGUAGE_SELECTOR': False,
 
-    # Write new CSM history to the extended table.
-    # This will eventually default to True and may be
-    # removed since all installs should have the separate
-    # extended history table.
-    'ENABLE_CSMH_EXTENDED': False,
+    # This may be removed when all installs have the separate extended history table.
```

Comment:
Oh - I see what you mean.  Sigh.  Can't I hope that someday there won't be differently configured installs?  I guess I should just remove the comment and then try to ride Dave's idea about removing CSMH altogether.